frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -357,20 +342,47 @@ class Worksheet extends React.Component {
         this.togglePopupNoEvent(cmd_type);
     };
 
+    addCopyBundleRowsCallBack = (tableID, callback) => {
```

Comment:
```suggestion
    addCopyBundleRowsCallback = (tableID, callback) => {
```

Can you also rename `CallBack` to `Callback` everywhere?